configs/experimental/imagecondition_zero123nerf.yaml

```diff
@@ -95,16 +98,14 @@ system:
   prompt_processor_type: "stable-diffusion-prompt-processor"
   prompt_processor:
     pretrained_model_name_or_path: "runwayml/stable-diffusion-v1-5"
-    prompt: "a DSLR photo of a delicious hamburger"
+    prompt: ???
 
   guidance_type: "stable-diffusion-guidance"
   guidance:
     pretrained_model_name_or_path: "runwayml/stable-diffusion-v1-5"
-    guidance_scale: 7.5
-    min_step_percent: 0.2
-    # min_step_percent: [0, 0.66, 0.33, 2000]  # (start_iter, start_val, end_val, end_iter)
-    max_step_percent: 0.6
-    # max_step_percent: [0, 0.98, 0.66, 2000]
+    guidance_scale: 100
+    min_step_percent: [0, 0.4, 0.2, 200]  # (start_iter, start_val, end_val, end_iter)
```

Comment:
Not for this PR, but for a later one... maybe try `min_step_percent: [0, 0.4, 0.02, 200]`... that might help with minor details

Reply:
Sure!